superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -376,11 +377,27 @@ class Header extends React.PureComponent {
     this.setState({ showingReportModal: false });
   }
 
-  handleReportClick() {
+  reportModal() {
     const attachedReportExists = this.props.report?.count > 0;
-    if (!attachedReportExists) {
-      this.showReportModal();
-    }
+    return !attachedReportExists ? (
```

Comment:
can you flip the return logic so that you can lead with the positive statement rather than negative? Just for ease of readability.

Reply:
yes, that makes so much sense.